crates/matrix-sdk-base/src/client.rs

```diff
@@ -1131,6 +1135,22 @@ impl BaseClient {
         }
     }
 
+    /// Get to-device requests that will share the details of a room key history
+    /// bundle with a user.
+    #[cfg(feature = "e2e-encryption")]
+    pub async fn share_room_key_bundle_data(
```

Comment:
Do we need this method? You're getting the `OlmMachine` in the main crate anyways, a couple of lines before this method gets called.

Reply:
Fixed in 4ded1e5ec